Add str as accepted type for resampling destinations (#3218)
pytroll/satpy: doc/source/config.rst

1. Create a directory to store your new custom YAML configuration files.
   The files for each component will go in a subdirectory specific to that
   component (ex. ``composites``, ``enhancements``, ``readers``, ``writers``).

>    A custom ``areas.yaml`` to configure area definitions should stay in the main directory.

Reviewer: ```suggestion
   A custom ``areas.yaml`` to configure area definitions can also be placed in
   the root of this directory.
```
